dist.ini

```
@@ -12,7 +12,7 @@ version = 0.002
 
 [Alien]
 :version = 0.023
-repo = ftp://ftp.astron.com/pub/file/
+repo = http://ftp.astron.com/pub/file/
```

```suggestion
repo = https://astron.com/pub/file/
```
:point_up: looks like this can be fetched from `https`